### PR TITLE
ani-cli: Add version 1.6

### PR DESCRIPTION
**Pull request by RaynardGerraldo**
ani-cli : A cli to browse and watch anime. This tool scrapes the site gogoanime.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ani-cli.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.6",
+    "description": "A cli to browse and watch anime. This tool scrapes the site gogoanime.",
+    "homepage": "https://github.com/pystardust/ani-cli",
+    "license": "GPL-3.0",
```
Comment (rashil2000) [inline bucket/ani-cli.json:5]:
```suggestion
    "license": "GPL-3.0-or-later",
```

File: bucket/ani-cli.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.6",
+    "description": "A cli to browse and watch anime. This tool scrapes the site gogoanime.",
```
Comment (rashil2000) [inline bucket/ani-cli.json:3]:
```suggestion
    "description": "A CLI to browse and watch anime. This tool scrapes the site gogoanime.",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by RaynardGerraldo** [comment]
Thanks @rashil2000

**Pull request merged**